src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    blue,
+    red,
+    white,
+    green,
+    purple,
```

Comment:
```suggestion
public enum Colors {
    BLACK,
    BLUE,
    RED,
    ...
```

we should use caps for enum elements